pkg/domainservice/maintenanceUseCase.go

```diff
@@ -0,0 +1,15 @@
+package domainservice
+
+// MaintenancePageModel contains data that gets displayed when the maintenance mode is active.
+type MaintenancePageModel struct {
+	Title string
+	Text  string
+}
```

Comment:
This is part of the adapter interface so it should be in domainservice/adapterInterfaces.go